Model/Transaction/Backfill.php

```diff
@@ -110,7 +110,7 @@ public function __construct(
         $this->transactionFactory = $transactionFactory;
         $this->orderFactory = $orderFactory;
         $this->refundFactory = $refundFactory;
-        $this->logger = $logger;
+        $this->logger = $logger->force();
```

Comment:
@JohnMTrimbleIII Do we need to set the filename here to `TaxjarConfig::TAXJAR_TRANSACTIONS_LOG`?